### DISCUSSION TITLE
Is there a maximum number of requests that can be placed on the hashtag API?

### DISCUSSION THREAD

**Discussion by jjzet** [davidteather/TikTok-Api#938] [Q&A]
I am running the following code to retrieve account info by hashtag, however it seems the maximum number of responses I can get on any hashtag is 500 or so. 

Is there a way I can request more? Say...10K lines of account info? Appreciate your help!

```
from TikTokApi import TikTokApi
import pandas as pd

hashtag = "ugc"
count = 10000

with TikTokApi() as api:
    tag = api.hashtag(name=hashtag)
    print(tag.info())

    lst = []
    for video in tag.videos(count=count):
        print(video.author.as_dict)
```

**Comment by davidteather** [comment]
Yeah most methods seem to have a maximum amount you can look into them until they (probably) require user login on the website itself.
